src/UI/Core/ListBox.cpp

```diff
@@ -280,19 +280,19 @@ void ListBox::update()
 	r.clipRect(rect().x() - 1, rect().y(), rect().width() + 1, rect().height() + 1);
 
 	// draw boundaries of the widget
-	r.drawBox(rect().x(), rect().y(), mItemWidth, rect().height(), 0, 0, 0, 100);
-	r.drawBoxFilled(rect().x(), rect().y(), mItemWidth, rect().height(), 0, 85, 0, 220);
+	r.drawBox(rect().x(), rect().y(), static_cast<float>(mItemWidth), rect().height(), 0, 0, 0, 100);
+	r.drawBoxFilled(rect().x(), rect().y(), static_cast<float>(mItemWidth), rect().height(), 0, 85, 0, 220);
 
-	// Highlight currently selected file
-	int itemY = rect().y() + (mCurrentSelection * mLineHeight) - mCurrentOffset;
-	r.drawBoxFilled(rect().x(), itemY, mItemWidth, mLineHeight, mHighlightBg.red(), mHighlightBg.green(), mHighlightBg.blue(), 80);
+	// Highlight currently selected item
+	float itemY = rect().y() + static_cast<float>((mCurrentSelection * mLineHeight) - mCurrentOffset);
```

Comment:
I need to reexamine why I chose `float` as the interface preference here. It's been awhile since I really thought about it but it stems from OpenGL's liberal use of float in its API so I figured why not stick with that, right?

The other part of that is for the use cases where someone may not want pixel alignment -- e.g., a slower crawl across a screen. It does result in texture filtering but that may be desirable in those cases. As stated, outside the scope of this PR but something to consider for future implementations of `NAS2D::RendererOpenGL`.